crates/trigger-http/src/lib.rs

```diff
@@ -706,7 +706,35 @@ impl OutboundWasiHttpHandler for HttpRuntimeData {
             return Self::chain_request(data, request, component_id);
         }
 
-        wasmtime_wasi_http::types::default_send_request(data, request)
+        // TODO: This is a temporary workaround to make sure that outbound task is instrumented.
+        // Once Wasmtime gives us the ability to do the spawn ourselves we can just call .instrument
+        // and won't have to do this workaround.
+        let response_handle = wasmtime_wasi_http::types::default_send_request(data, request)?;
+        let response = data.table().get_mut(&response_handle)?;
+        *response = match std::mem::replace(response, HostFutureIncomingResponse::Consumed) {
+            HostFutureIncomingResponse::Pending(handle) => {
+                HostFutureIncomingResponse::Pending(wasmtime_wasi::preview2::spawn(
+                    async move {
+                        let res: std::prelude::v1::Result<
+                            std::prelude::v1::Result<
```

Comment:
Yes, `Result` is `anyhow::Result`, but it will work anyway
```suggestion
                        let res: Result<
                            Result<
```